_i18n/en.yml

```diff
@@ -214,12 +218,64 @@ help_page:
   p_1: Browse common topics
   title: How can we help?
 help_pages:
+  get-started:
+    overview: |-
+      Login.gov is the public's one account and password for government. Login.gov is a shared service and trusted by government agencies. With one login.gov account you can access applications from participating government partners.
+
+      Create your login.gov account
+      * Learn about authentication options
+      * Why you are being asked to use login.gov
+
+      ## Common troubleshooting topics {#help-question-list}
```

Comment:
Now that @aduth and @zachmargolis showed me the markdown ID, we can use this ID to style the fancy question bullets. See sass updates.

Reply:
WDYT of making this a class instead of an ID? In case we ever want more than one of these on a page?

Reply:
> WDYT of making this a class instead of an ID? In case we ever want more than one of these on a page?

Also, it won't work as-is since the styles are only in effect for the class, not ID.

Reply:
I looked it up, but I don't see markdown support for classes this way. If you have a reference please lmk.

Reply:
Here's the example of Kramdown supporting classes:

https://about.gitlab.com/blog/2016/07/19/markdown-kramdown-tips-and-tricks/#applying-classes

I tried this on a random page and it worked (for block-level element, the attribute need to go on the line below)

```md
## Foobar

some text

## Foobar with class
{: .foobar-class}

some text
```

this is the HTML it turned into:


 
```html
<h2 id="foobar">Foobar</h2>
<p>some text</p>
<h2 class="foobar-class" id="foobar-with-class">Foobar with class</h2>
<p>some text</p>
```

Reply:
Thanks! I'm going to implement with the french translation PR. Merging this now for testing.

Reply:
Sounds good!